luthier/config/src/utils/dimensionUtils.lua

```diff
@@ -55,7 +55,7 @@ function M.build_dimensions_config(dimensions)
         dim_copy.apiName = dim_copy.apiName or name
         dim_copy.longName = dim_copy.longName or name
         dim_copy.description = dim_copy.description or name
-        table.insert(configuration, dim_copy)
+        configuration[ dim_copy.apiName ] = dim_copy
```

Comment:
In Lua, you don't typically put spaces in table lookups. So this
should be:
```lua
configuration[dim_copy.apiName] = dim_copy
```

Reply:
Can I make this change here in the PR or should I redo it on my branch and submit a new PR?

Reply:
The expected workflow is that you add additional commits to this PR to address people's comments. So the flow is:

1. Add PR.
2. People --stare in awe at your incredible code-- make comments.
3. Add commits to the PR that address those commits.
4. If reviews approve, merge the PR, else go to 2.